exposee-app/Routes/users.js

```diff
@@ -0,0 +1,86 @@
+import express from 'express';
+import bcrypt from 'bcrypt';
+import { User } from '../models/users.js';
+import { Op } from 'sequelize';
+
+const router = express.Router();
+
+
+router.get ('/user', async(req,res)=> {
+  
+  
+  const users = await User.findAll();
+  console.log(users)
+  res.status(200).json(users);
+  
+     });
```

Comment:
todo: add a linter to your project so you can lint your code and avoid formatting issues like this